.github/workflows/mirror-tooling-images.yml

```diff
@@ -35,6 +35,13 @@ jobs:
           - "mcr.microsoft.com/azurelinux/base/core:3.0"
           - "mcr.microsoft.com/cbl-mariner/base/core:2.0"
           - "mcr.microsoft.com/cbl-mariner/base/core:1.0"
+          - "registry.opensuse.org/opensuse/leap:15.6"
+          - "registry.opensuse.org/opensuse/leap:16.0"
+          - "registry.opensuse.org/opensuse/leap:latest"
+          - "registry.opensuse.org/opensuse/tumbleweed:latest"
+          - "registry.suse.com/bci/bci-base:15.6"
+          - "registry.suse.com/bci/bci-base:15.7"
+          - "registry.suse.com/bci/bci-base:latest"
 
```

Comment:
is there an existing or custom image we can add to the integration test workflow as well?

Reply:
Not sure if I understood what you mean with `existing or custom image`. Let me check the e2e workflow and see if I can figure some logic there.

Reply:
I've added the tests for single and multi arch platforms. The failing test is unrelated to this PR, but I cannot re-trigger it.

Reply:
Sorry, apparently there is an error with a BCI e2e test that I need to fix.

Reply:
@ashnamehrotra I've add the e2e tests, but one of them keeps failing -> https://github.com/project-copacetic/copacetic/actions/runs/18202094346/job/51823355712?pr=1294. I'm trying to find the issue.

Reply:
It seems that the errors are because copa isn't find its expected annotation in the patched image - https://github.com/project-copacetic/copacetic/actions/runs/18202094346/job/51823355764?pr=1294#step:16:10952.

Reply:
I fixed the added e2e tests and they are passing now. The failed workflow is unrelated and a retrigger should fix it.